.github/workflows/cypress-tests.yml

```diff
@@ -17,7 +17,8 @@ jobs:
     # Platform recommended in cypress-io/github-action docs.
     runs-on: ubuntu-16.04
     # Manually skip this 10+ minute build based on commit message.
-    if: "!contains(github.event.head_commit.message, '[skip ci]')"
+    # Translation: If "skip ci" substring NOT found in push message or title of PR (hack).
+    if: "! (contains(github.event.head_commit.message, '[skip ci]') || contains(github.event.pull_request.title, '[skip ci]'))"
```

Comment:
This was simply cherry-picked from another pending PR, to help conserve build hours.